src/cljs/ataru/hakija/application_form_components.cljs

```diff
@@ -461,42 +461,42 @@
-  (let [id        (str component-id "-upload-button")
-        language  @(subscribe [:application/form-language])
-        on-change (fn [event]
-                    (.preventDefault event)
-                    (let [file-list  (or (some-> event .-dataTransfer .-files)
-                                         (.. event -target -files))
-                          files      (->> (.-length file-list)
-                                          (range)
-                                          (map #(.item file-list %)))
-                          file-sizes (map #(.-size %) files)]
-                      (if (some #(> % max-attachment-size-bytes) file-sizes)
-                        (dispatch [:application/show-attachment-too-big-error component-id])
-                        (dispatch [:application/add-attachments field-descriptor component-id attachment-count files]))))]
-    (fn [field-descriptor component-id attachment-count]
-      [:div.application__form-upload-attachment-container
-       [:input.application__form-upload-input
-        {:id        id
-         :type      "file"
-         :multiple  "multiple"
-         :key       (str "upload-button-" component-id "-" attachment-count)
-         :on-change on-change}]
-       [:label.application__form-upload-label
-        {:for id}
-        [:i.zmdi.zmdi-cloud-upload.application__form-upload-icon]
-        [:span.application__form-upload-button-add-text (case language
-                                                          :fi "Lisää liite..."
-                                                          :en "Upload attachment..."
-                                                          :sv "Ladda upp bilagan...")]]
-       (let [file-size-info-text (case language
-                                   :fi "Tiedoston maksimikoko on 10 MB"
-                                   :en "Maximum file size is 10 MB"
-                                   :sv "Den maximala filstorleken är 10 MB")]
-         (if @(subscribe [:state-query [:application :answers (keyword component-id) :too-big]])
-           [:span.application__form-upload-button-error.animated.shake file-size-info-text]
-           [:span.application__form-upload-button-info file-size-info-text]))])))
+  (let [id       (str component-id "-upload-button")
+        language @(subscribe [:application/form-language])]
+    [:div.application__form-upload-attachment-container
+     [:input.application__form-upload-input
+      {:id        id
+       :type      "file"
+       :multiple  "multiple"
+       :key       (str "upload-button-" component-id "-" attachment-count)
+       :on-change (partial upload-attachment field-descriptor component-id attachment-count)}]
+     [:label.application__form-upload-label
+      {:for id}
+      [:i.zmdi.zmdi-cloud-upload.application__form-upload-icon]
+      [:span.application__form-upload-button-add-text (case language
+                                                        :fi "Lisää liite..."
```

Comment:
Not a new thing in this PR, but these really should live in the localization files...

Reply:
Absolutely. I propose that we fix this globally at some point in the near future.